automation/terraform/testnets/mainnet/main.tf

```diff
@@ -0,0 +1,133 @@
+terraform {
+  required_version = ">= 0.14.0"
+  backend "s3" {
+    key     = "terraform-mainnet.tfstate"
+    encrypt = true
+    region  = "us-west-2"
+    bucket  = "o1labs-terraform-state"
+    acl     = "bucket-owner-full-control"
+  }
+}
+
+provider "aws" {
+  region = "us-west-2"
+}
+
+provider "google" {
+  alias   = "google-us-east4"
+  project = "o1labs-192920"
+  region  = "us-east4"
+  zone    = "us-east4-b"
+}
+
+provider "google" {
+  alias   = "google-us-east1"
+  project = "o1labs-192920"
+  region  = "us-east1"
+  zone    = "us-east1-b"
+}
+
+provider "google" {
+  alias   = "google-us-central1"
+  project = "o1labs-192920"
+  region  = "us-central1"
+  zone    = "us-central1-c"
+}
+
+
+variable "whale_count" {
+  type = number
+
+  description = "Number of online whales for the network to run"
+  default     = 0
+}
+
+variable "fish_count" {
+  type = number
+
+  description = "Number of online fish for the network to run"
+  default     = 0
+}
+
+variable "seed_count" {
+  default     = 6
+}
+
+locals {
+  testnet_name = "mainnet"
+  coda_image = "TODO" # TODO
+  coda_archive_image = "TODO" # TODO
+  seed_region = "us-east4"
+  seed_zone = "us-east4-b"
+
+  # replace with `make_report_discord_webhook_url = ""` if not in use (will fail if file not present)
+  make_report_discord_webhook_url = <<EOT
+    ${file("../../../discord_webhook_url.txt")}
+  EOT
+
+  # replace with `make_report_accounts = ""` if not in use (will fail if file not present)
+  # make_report_accounts = <<EOT
+  #   ${file("../../../${local.testnet_name}-accounts.csv")}
+  # EOT
+  make_report_accounts = ""
+}
+
+module "mainnet" {
+  providers = { google.gke = google.google-us-east4 }
```

Comment:
We should use us-east-1 for mainnet or set up beefier nodes in another region, I want  to go a bit overkill on resource allocation just so we can handle a lot of load (and connections) if need be.

Reply:
And to match, set the cpu and memory requests for seeds to 22 cores and 48gb ram or something (a few cores/gb less than the max capacity of the nodes)